app/lib/image_pipeline_service.rb

```diff
@@ -72,12 +114,42 @@ def self.create_expression_parameters_object(cluster_file, matrix_file)
     RenderExpressionArraysParameters.new(parameters)
   end
 
+  # create a ImagePipelineParameters object to pass to IngestJob
+  #
+  # * *params*
+  #   - +study+       (Study) => study to generate images in
```

Comment:
Since we're aligning indentation:

```suggestion
  #   - +study+        (Study) => study to generate images in
```

Reply:
[69c0769](https://github.com/broadinstitute/single_cell_portal_core/pull/1663/commits/69c0769a4cb881b9c814c8e159f538b10e4298ba)